server/app/controllers/admin/AdminProgramStatusesController.java

```diff
@@ -128,32 +128,41 @@ private ErrorAnd<ProgramDefinition, CiviFormError> createOrEditStatusFromFormDat
               .setLocalizedStatusText(LocalizedStrings.withDefaultValue(formData.getStatusText()));
       if (!formData.getEmailBody().isBlank()) {
         newStatusBuilder =
-            newStatusBuilder
-                .setEmailBodyText(Optional.of(formData.getEmailBody()))
-                .setLocalizedEmailBodyText(
-                    Optional.of(LocalizedStrings.withDefaultValue(formData.getEmailBody())));
+            newStatusBuilder.setLocalizedEmailBodyText(
+                Optional.of(LocalizedStrings.withDefaultValue(formData.getEmailBody())));
       }
       return service.appendStatus(program.id(), newStatusBuilder.build());
     }
     return service.editStatus(
         program.id(),
         formData.getConfiguredStatusText(),
         (existingStatus) -> {
+          // TODO(#2752): Disable the English translations UI
```

Comment:
Gotcha. I was hoping the formatter would catch/fix that if we want to approach the 100-character limit. I generally try to wrap based on the code around it since I find it more readable.

I very much don't have a strong enough opinion here to push back though :).

Reply:
Ah interesting pattern.  I believe the general style guides prefer fitting to 100 as a default for less vertical space.